### DISCUSSION TITLE
CERTIFICAÇÃO DE SOFWARE EM ANGOLA

### DISCUSSION THREAD

**Discussion by Domingosantonio** [assoft-portugal/SAF-T-AO#200] [General]
Saudações pessoal,

Alguém que já tenha feito a certificação do software na AGT, pode pff dar algum input sobre o processo?
É que já vai a 2 meses que eu fiz a solicitação no Portal do Produtor de Software [https://invoice.minfin.gov.ao/portalSupplier/#/login], até agora não recebo nenhum sinal por parte da AGT.

Obrigado desde já,

**Comment by ravunana** [comment]
Saudaçoes Domingos, também estou neste processo, fiz minha solicitação em Junho até o dado momento não tenho nenhum feedback por parte da AGT, como a sua Mensagem é de Maio, quero saber se tens já um feedbackda AGT?

Obrigado desde já.

**Reply by Domingosantonio** [reply]
A solicitação foi feita em Abril 😥, mas até agora não recebi nenhum feedback por parte da AGT.

**Reply by FernandoManuel6107** [reply]
@cryptolopes podes ajudar? Quase que estou também na mesma situação, o meu caso é uma renovação.

**Reply by cryptolopes** [reply]
Em ambos os casos devem aguardar resposta por parte da AGT, o que não impede que se vá insistindo junto dos seus serviços.
